### PR TITLE
Remove mentions of "eth2" [Fixes #421]

### LINKED ISSUE #421: Replace copy specific to Eth2 round
Refactor all hard-coded references in favor of more flexible solution for any clr.fund instance.

What should approach here be? Simply drop "Eth2" copy?
- https://github.com/ethereum/clrfund/blob/develop/vue-app/src/views/Landing.vue#L12
  - e.g. "Send your favorite Eth2 projects to the moon!" --> "Send your favorite projects to the moon!"
- https://github.com/ethereum/clrfund/blob/develop/vue-app/src/components/CriteriaModal.vue#L53
- https://github.com/ethereum/clrfund/blob/develop/vue-app/src/router/index.ts#L38
- https://github.com/ethereum/clrfund/blob/develop/vue-app/src/components/RoundInformation.vue#L10
- https://github.com/ethereum/clrfund/blob/develop/vue-app/src/views/TransactionSuccess.vue#L41

### PR DESCRIPTION
**Pull request by wackerow**
### Description
- Removes mentions of "eth2" from copy
- Extracted round criteria into own `criteria.ts` file to more easily update these criteria for future rounds

### Related issues
#421

### CODE REVIEW

File: vue-app/src/components/CriteriaModal.vue [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <template>
   <div class="wrapper">
-    <div class="modal-background" @click="toggleCriteria" />
+    <div class="modal-background" @click="$emit('close')" />
     <div class="container">
       <div>
         <div class="flex-row">
           <h2>Round criteria</h2>
-          <div class="close-btn" @click="toggleCriteria">
+          <div class="close-btn" @click="$emit('close')">
```
Comment (pettinarip) [inline vue-app/src/components/CriteriaModal.vue:8]: 💪🏼 events ftw

File: vue-app/src/components/CriteriaModal.vue [hunk 2]
```diff
@@ -40,50 +40,14 @@
...
-    {
-      emoji: '💻',
-      criteria: 'No clients',
-      description:
-        'Client teams are so important but this round of funding is focused on supporting other parts of the ecosystem.',
-    },
-  ]
+  get criteria(): Criterion[] {
+    return criteria
+  }
```
Comment (pettinarip) [inline vue-app/src/components/CriteriaModal.vue]: Really like this refactor, gj.

**Review by pettinarip — APPROVED** [review]
Great! 🚀

**Review by samajammin — APPROVED** [review]
👍

**Pull request merged**